feature_server/run.py

```diff
@@ -57,6 +57,10 @@ def choose_path(base,top):
 
 # add it to the path so we can import scripts
 sys.path.append(config_dir)
+# add our package to path too so scripts can import `feauture_server/`
```

Comment:
is this necessary? I would have thought if installed as a python package (or running in a virtualenv), `feature_server` should be available to import regardless.